code/index.html

```diff
@@ -3,28 +3,182 @@
+                </p>
+              </div>
+
+          </div>  
+
+          <div class="accordion-element">
+        
+            <h1 id="accordion-header-2" class="accordion-header">
+            Tickets
+            </h1>
+       
+              <div class="accordion-text-container">
+                <p class="accordion-text"> 
+                  Lorem, ipsum dolor sit amet consectetur adipisicing elit. 
+                  Officia, voluptatibus? Doloribus consequatur facilis neque 
+                  a met perspiciatis cumque ullam repudiandae eaque dicta nulla 
+                  reprehenderit, debitis odit facere magni minima, omnis illum.
+                </p>
+              </div>
+
+          </div>  
+     
+
+          <div class="accordion-element">
+        
+            <h1 id="accordion-header-3" class="accordion-header">
+            Rules
+            </h1>
+       
+              <div class="accordion-text-container">
+                <p class="accordion-text"> 
+                  Lorem, ipsum dolor sit amet consectetur adipisicing elit. 
+                  Officia, voluptatibus? Doloribus consequatur facilis neque 
+                  a met perspiciatis cumque ullam repudiandae eaque dicta nulla 
+                  reprehenderit, debitis odit facere magni minima, omnis illum.
+                </p>
+              </div>
 
-      <div class="question" id="section2">Section 2</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+          </div>
+
+
+          <div class="accordion-element">
+        
+            <h1 id="accordion-header-4" class="accordion-header">
+            The price
+            </h1>
+       
+              <div class="accordion-text-container">
+                <p class="accordion-text"> 
+                  Lorem, ipsum dolor sit amet consectetur adipisicing elit. 
+                  Officia, voluptatibus? Doloribus consequatur facilis neque 
+                  a met perspiciatis cumque ullam repudiandae eaque dicta nulla 
+                  reprehenderit, debitis odit facere magni minima, omnis illum.
+                </p>
+              </div>
+
+          </div>
+      
       </div>
-    </section>
-    <!-- dont forget to link the javascript file here! -->
-  </body>
+    
+      <!-- Signup form -->
+      <div class= "flex-child form">
+        <form 
+        class="form-container" 
+        action="http://httpbin.org/anything" 
+        method="POST"
+      >
+
+        <div class="form-group">
+          <label for="firstName">First name *</label>
+          <input 
+          id="firstName" 
+          type="text" 
+          name="firstNameValue"
+          required>
+          
+        </div>
+        
+        <div class="form-group">
+          <label for="lastName"> Last name *</label>
+          <input 
+          id="lastName" 
+          type="text"
+          name="lastNameValue"
+          required>
```

Comment:
Perfect use of the label with the 'for' attribute an also nice to see some simple validation! BRAVO 🙌